test/setup/deployers/operator-openshift.ts

```diff
@@ -8,10 +10,30 @@ export const operatorDeployer: IDeployer = {
 async function deployKubernetesMonitor(
   _imageOptions: IImageOptions,
 ): Promise<void> {
-  await kubectl.applyK8sYaml('./test/fixtures/operator/operator-source.yaml');
+  const overriddenOperatorSource = 'snyk-monitor-operator-source.yaml';
+  createTestOperatorSource(
+    overriddenOperatorSource,
+    process.env['QUAY_USERNAME']!,
+  );
+
+  await kubectl.applyK8sYaml(overriddenOperatorSource);
   await kubectl.applyK8sYaml('./test/fixtures/operator/installation.yaml');
   // Await for the Operator to become available, only then
   // the Operator can start processing the custom resource.
   await kubectl.waitForDeployment('snyk-operator', 'snyk-monitor');
   await kubectl.applyK8sYaml('./test/fixtures/operator/custom-resource.yaml');
 }
+
+function createTestOperatorSource(newYamlPath: string, username: string): void {
+  console.log('Creating YAML OperatorSource...');
+  const originalDeploymentYaml = readFileSync(
```

Comment:
Does the blocking `readFileSync` method here have advantages over something like `await util.promisify(readFile)....`?

Reply:
It is in a test file/suite so being async here is not super important, and we know this file specifically is tiny